workshop/Lab1/README.md

```diff
@@ -0,0 +1,106 @@
+guestbook   NodePort   10.10.10.253   <none>        3000:31208/TCP   1m
+```
+
+We can see that our `<nodeport>` is `31208`. We can see in the output the port mapping from 3000 inside 
+the pod exposed to the cluster on port 31208. This port in the 31000 range is automatically chosen, 
+and could be different for you.
+
+`guestbook` is now running on your cluster, and exposed to the internet. We need to find out where it is accessible.
+The worker nodes running in the container service get external IP addresses.
+Run `bx cs workers <name-of-cluster>`, and note the public IP listed on the `<public-IP>` line.
+
+```
+$ bx cs workers osscluster
+OK
+ID                                                 Public IP        Private IP     Machine Type   State    Status   Zone    Version  
+kube-hou02-pa1e3ee39f549640aebea69a444f51fe55-w1   173.193.99.136   10.76.194.30   free           normal   Ready    hou02   1.5.6_1500*
+```
+
+We can see that our `<public-IP>` is `173.193.99.136`.
+   
+Now that you have both the address and the port, you can now access the application in the web browser
+at `<public-IP>:<nodeport>`. In the example case this is `173.193.99.136:31208`.
+   
+You've now deployed the basic version of the application. If you've worked through the [kube101 workshop](https://github.com/IBM/kube101/tree/master/workshop)
+this should look fairly familiar. We've just deployed Guestbook without an attached backing store, so it has defaulted to storing state in memory. You can verifiy
+this by following the `/info` link on the front page, and it should display that the application is storing data in memory.
+
+Test out the application by entering a few entries. Then, delete the Guestbook pod running in your cluster. Run `kubectl get pods` and you should get something
+like this:
+```
+$ kubectl get pods
+NAME                         READY     STATUS    RESTARTS   AGE
+guestbook-7b79469f5f-2tjhm   1/1       Running   0          14m
+```
+
+Then run `kubectl delete pod guestbook-7b79469f5f-2tjhm`. This will delete the container where your instance of Guestbook is actually running, but the Guestbook
+deployment should see this and start up another one. After a few seconds, if you run `kubectl get pods` again, you should see a new pod has been started:
+```
+> kubectl get pods
+NAME                         READY     STATUS    RESTARTS   AGE
+guestbook-7b79469f5f-p9p8t   1/1       Running   0          11s
+```
+
+If we go back and refresh Guestbook in our browser, we can see that the entries 
+we previously created are gone. This is obviously not ideal. We would like data
+entered into our application to persist beyond the lifetime of individual containers.
+In [the next lab of this course](../Lab2/README.md), we'll learn to manually provision
+and connect some stable storage to persist our application data.
+
+When you're all done, you can either use this deployment in the next lab,
```

Comment:
Let's just assume they're going to keep going, so just mention they'll use this deployment in the next lab and not the option of deleting it.

Reply:
And for consistency, let's have the same type of footer at the end of each lab with an href to the next one.